### ISSUE TITLE
Do not use an absolute URL with the domain in the website top navigation bar

### ISSUE THREAD

**Issue by carolynvs** [getporter/porter#2191]
When I am using getporter.org and I click on a link in the top navigation bar , it is taking me to porter.sh instead of staying on the getporter.org domain.

Let's update the top navigation links to not include the full domain, and instead use links like "/docs", and "/install".

**Comment by carolynvs** [comment]
@hrittikhere That would be really helpful, thank you!

**Comment by hrittikhere** [comment]
@carolynvs we need to close this issue as https://github.com/getporter/porter/pull/2195 is merged 😁

**Comment by carolynvs** [comment]
Looks like it didn't close because the issue had a link to the closed issue instead of just the issue number

```
Closes https://github.com/getporter/porter/issues/2191
```

vs

```
Closes #2191
```


In the future if you use the Closes #NUMBER format it should close the issue automatically
